app/src/main/java/android/andela/com/kenyajavadevs/model/GithubUsersResponse.java

```diff
@@ -7,7 +7,7 @@
 
 public class GithubUsersResponse {
 
-    @SerializedName("")
+    @SerializedName("items")
```

Comment:
Use more appropriate naming. @cmplx-xyttmt

Reply:
"items" is the key that comes back from GitHub containing the search results. However, I store the actual items in an array called "githubUsers".